metron-extensions/metron-parser-extensions/metron-parser-bro-extension/metron-parser-bro/src/main/config/zookeeper/enrichments/bro.json

```diff
@@ -0,0 +1,17 @@
+{
+    "enrichment" : {
```

Comment:
actually, we don't have defaults as is, I added them.  And the archetype produces them for you